Link to model docs from relationship limits.

johngallardo opened the pull request:
Reference our modeling best practices in regards to the limits on maximum number of outgoing and incoming relationships from a single twins. Customers will have more success in many cases by modeling these as properties rather than relationships.

johngallardo committed:
Link to model docs from relationship limits.

Reference our modeling best practices in regards to the limits on maximum number of outgoing and incoming relationships from a single twins. Customers will have more success in many cases by modeling these as properties rather than relationships.

dwappa commented on includes/digital-twins-limits.md:
```diff
@@ -15,8 +15,8 @@ The table below lists the functional limits of Azure Digital Twins.
 | Digital twins | Number of twins in an Azure Digital Twins instance | 200,000 | Yes |
-| Digital twins | Number of incoming relationships to a single twin | 5,000 | No |
-| Digital twins | Number of outgoing relationships from a single twin | 5,000 | No |
+| Digital twins | Number of incoming relationships to a single twin. See [Modeling Best Practices](../articles/digital-twins/concepts-models.md#best-practices-for-designing-models) for alternatives. | 5,000 | No |
```
Bailey I'm wondering to keep the table concise should this be a "note" on these two items which is expanded below?  Just a stylistic comment... maybe we move the note to the "Adjustable: no" since that is when it would be relevant.

johngallardo replied:
See Update.

baanders reviewed (commented):
@johngallardo  Thank you for adding this info! 
I also like Dan's suggestion about moving it to a note below the chart--would keep the table format concise, avoid duplicating the text, and enable us to write a bit more description (I like the clarity you added in your PR description that the reasoning for visiting the models doc is that sometimes it can be better to model them as properties rather than relationships).

What do you think about this edit?

johngallardo committed:
Update digital-twins-limits.md

Moved pointer to modeling recommendations to table header rather than inline within the table.

johngallardo committed:
Update digital-twins-limits.md

Remove punctuation that snuck in.

johngallardo committed:
Update digital-twins-limits.md

Moving recommendation to `[!TIP]` content.

baanders reviewed (approved):
Thank you @johngallardo, looks good!

ktoliver committed:
public repo edit/Microsoft style
